docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -71,16 +71,6 @@ __Description__: changes tunnel sharing permissions so that all users in an orga
 __Shorthand__: `-s`
 
 
----
-### `--tunnel-name`
-<p><small>| OPTIONAL | STRING |</small></p>
-
-__Description__: Assigns a name to a Sauce Connect Proxy tunnel. Strongly recommended for best performance. It can also assign a name to a group of tunnels in the same [High Availability pool]((/secure-connections/sauce-connect/setup-configuration/high-availability), when used with [`--tunnel-pool`](#--tunnel-pool). Must be in ASCII format.
-
-Future jobs will use this tunnel only when explicitly specified by the [`tunnelName`](/dev/test-configuration-options/#tunnelname) in your test capabilities. To learn about the syntax for setting this as a capability, see [Test Configuration Options](/dev/test-configuration-options).<br/>
-__Shorthand__: n/a
-
-
```

Comment:
@mdsauce @waggledans  So the `--tunnel-identifier` CLI flag was resurrected? The last time I used it in 4.7.1, it failed. But I tested now and looks like now both `--tunnel-identifier` and `--tunnel-name` work?

cc: @sweeneyskirt-sl